pkgs/top-level/all-packages.nix

```diff
@@ -9915,7 +9915,10 @@ in
 
   cmake_2_8 = callPackage ../development/tools/build-managers/cmake/2.8.nix { };
 
-  cmake = libsForQt5.callPackage ../development/tools/build-managers/cmake { };
+  cmake = libsForQt5.callPackage ../development/tools/build-managers/cmake {
+    # circular dependency
+    libarchive = libarchive.override { zstd = null; };
```

Comment:
Do you think it’s at all likely that somebody might want to use CMake with zstd? If so, you could instead pass zstd a custom CMake with no zstd support, and leave the cmake attribute pointing to one with zstd enabled. I doubt any of this takes very long to compile.

Reply:
Done. The CMake hash changing is a mass rebuild, though.